addOns/fuzzdb/CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ All notable changes to this add-on will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
+## [9] - 2022-03-18
+- Updated RAFT lists based on more recent SecLists contributions
```

Comment:
This should be in the `### Changed` section just below.

Reply:
And the previous line (i.e. `## [9]…`) removed.

Reply:
updated!